modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+    LINFO("Loading image collections to " + id);
+
+    // Load the collections here because here we know that the browser can execute 
+    // javascript
+    std::string root = "https://raw.githubusercontent.com/WorldWideTelescope/"
+                        "wwt-web-client/master/assets/webclient-explore-root.wtml";
+
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    if (module->getPair(id)) {
+        module->getPair(id)->hideChromeInterface(true);
+        module->getPair(id)->loadImageCollection(root);
+    }
+
+    return 0;
+}
+
+int startSetup(lua_State* L) {
+    // This is called when the sky_browser website is connected to OpenSpace
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::startSetup");
+
+    // Set all border colors to the border color in the master node
+    if (global::windowDelegate->isMaster()) {
+        SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+        std::vector<std::unique_ptr<TargetBrowserPair>>& pairs = module->getPairs();
+        for (std::unique_ptr<TargetBrowserPair>& pair : pairs) {
+            std::string id = pair->browserId();
+            glm::ivec3 color = pair->borderColor();
+            std::string script = fmt::format(
+                "openspace.skybrowser.setBorderColor('{}', {}, {}, {})", 
+                id, 
+                color.r, 
+                color.g, 
+                color.b
+            );
+            global::scriptEngine->queueScript(
+                script,
+                scripting::ScriptEngine::RemoteScripting::Yes
+            );
+        }
+    }
+   
+    // To ensure each node in a cluster calls its own instance of the wwt application
+    // Do not send this script to the other nodes
+    global::scriptEngine->queueScript(
+        "openspace.skybrowser.sendOutIdsToBrowsers();",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+    
+    return 0;
+}
+
+int sendOutIdsToBrowsers(lua_State* L) {
+    // This is called when the sky_browser website is connected to OpenSpace
+    ghoul::lua::checkArgumentsAndThrow(L, 0, "lua::sendOutIdsToBrowsers");
+
+    // Send out identifiers to the browsers
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    std::vector<std::unique_ptr<TargetBrowserPair>>& pairs = module->getPairs();
+    for (std::unique_ptr<TargetBrowserPair>& pair : pairs) {
+        pair->sendIdToBrowser();
+    }
+
+    return 0;
+}
+
+int initializeBrowser(lua_State* L) {
+    // Initialize browser with ID and its corresponding target
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::initializeBrowser");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+
+    LINFO("Initializing sky browser " + id);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* found = module->getPair(id);
+    if (found) {
```

Comment:
Instead of checking `if (found)` here (and silently ignore the cases where it is not), I would change to checking if it was not found and provide an error if it wasn't:

```
    if (!found) {
        return ghoul::lua::luaError(L, fmt::format("Could not find pair with identifier: {}"),  id);
    }
```
Or a similar message. 

This goes for all the other functions in this file that does a check like this.